### PR TITLE
Add the ability to create and send a voting data

### PR DESCRIPTION
**Pull request by MichaelKim20**
I added information to access Agora to `BOAClient`
I added the ability to create and send a voting data

Related to #17

### CODE REVIEW

File: src/modules/data/VoteData.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    index: number;
+}
+
+interface IVoteData
+{
+    inputs: Array<ITxInput>;
+    keys: Array<string>;
+    data: string;
+    address: string;
+}
```
Comment (TrustHenry) [inline src/modules/data/VoteData.ts:121]: Blank line insertion required

Comment (MichaelKim20) [reply src/modules/data/VoteData.ts]: I fixed it

**Comment by Geod24** [comment]
Needs a rebase

**Comment by MichaelKim20** [comment]
I rebased it

File: src/modules/data/VoteData.ts [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+/**
+ * The class that defines the vote data.
+ */
+export class VoteData
+{
+    /**
+     * An array of 1 or more UTXOs to be spent
+     */
+    public inputs: Array<TxInput>;
```
Comment (MichaelKim20) [inline src/modules/data/VoteData.ts:28]: When using an utxo hash, do the following:
It is necessary to confirm which is most appropriate.
public inputs: Array<Hash>;

**Pull request closed without merging**